### PR TITLE
[Documentation] How to file good SwiftPM bug

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
- https://bugs.swift.org/browse/SR-3243

### CODE REVIEW

File: Documentation/Resources.md [hunk 1]
```diff
@@ -32,9 +32,23 @@ User-to-user support for Swift Package Manager is available on [swift-users mail
...
-## Reporting Bugs
+## Reporting a good SwiftPM Bug
 
-Use the [Swift bug tracker](http://bugs.swift.org) to report bugs with Swift Package Manager.
+Use the [Swift bug tracker](http://bugs.swift.org) to report bugs with Swift Package Manager. Sign up if you haven't already and click the "Create" button to start filing an issue.  
+
+Fill the following fields:
+* `Summary`: One line summary of the problem you're facing  
+* `Description`: The complete description of the problem. Be specific and clearly mention the steps to reproduce the bug  
+* `Enviornment`: The Operating System, Xcode, Toolchain and `swift build` version (`$ swift build --version`)
```
Comment (neonichu) [inline Documentation/Resources.md]: `Environment`

Comment (neonichu) [inline Documentation/Resources.md]: Maybe we should also tell people how to obtain the other version information (e.g. `xcodebuild -version`)

Comment (ddunbar) [inline Documentation/Resources.md]: Agreed

**Comment by ddunbar** [comment]
LGTM w/ Boris's comment.

**Pull request merged**